### ISSUE TITLE
Export the `HelmetOptions` interface

### ISSUE THREAD

**Issue by poppa** [helmetjs/helmet#271]
It would be nice if you could export the `HelmetOptions` interface. It's not uncommon to want to propagate an interface in another larger interface.

```ts
export interface MyLargeConfig {
  // [...]
  helmetOptions?: ???
}
```

**Comment by EvanHahn** [comment]
Would something like this work for you (lifted from [this comment](https://github.com/helmetjs/helmet/issues/235#issuecomment-674016883))?

```typescript
import helmet = require('helmet');
const opts: Parameters<typeof helmet>[0] = {
  // strongly typed helmet options go here...
};
helmet(opts);
```

**Comment by poppa** [comment]
Thanks, that works indeed, but I don't know if it's a nice solution. It feels very indirect and more like a hack than a solution. 

But I'm good with it, and if you have good reasons for not exporting the options interface, it's fine by me :)

**Comment by EvanHahn** [comment]
I just released Helmet v5.0.0-beta.1 which addresses this. You can now `import helmet, { HelmetOptions } from 'helmet'`. Please give it a try with `npm install helmet@5.0.0-beta.1` and let me know if there are any problems! Feel free to reply here or email me@evanhahn.com with any feedback.
